class TestException(Exception):
    raise Exception("test exception")
